src/main/java/taxi/dao/impl/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,110 @@
+package taxi.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import taxi.dao.ManufacturerDao;
+import taxi.lib.Dao;
+import taxi.model.Manufacturer;
+import taxi.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String sqlQuery = "INSERT INTO manufacturer (manufacturer_name, manufacturer_country)"
```

Comment:
```suggestion
        String sqlQuery = "INSERT INTO manufacturer (name, country)"
```
You already know it is a manufacturer

Reply:
done

Reply:
done